lib/client.js

```diff
@@ -734,6 +734,10 @@ client.prototype.handleMessage = function handleMessage(message) {
 						var viewers = +message.tags["msg-param-viewerCount"];
 						this.emit("raided", channel, username, viewers, userstate);
 						break;
+					// Handle new chatter ritual
+					case "ritual":
+						this.emit("newchatter", channel, username, userstate, msg);
```

Comment:
There _may_ be other rituals added later, they're not all necessarily going to be `msg-param-ritual-name=new_chatter`. I suggest switching on `"msg-param-ritual-name"` for individual known events defaulting to a `"ritual"` event with the ritual name in the parameters.
```js
case "ritual":
	var ritualName = userstate["msg-param-ritual-name"];
	switch(ritualName) {
		case "new_chatter":
			this.emit("newchatter", channel, username, userstate, msg);
			break;
		default:
			this.emit("ritual", ritualName, channel, username, userstate, msg);
			break;
	}
```